packages/toolpad-core/src/Account/AccountPreview.tsx

```diff
@@ -58,6 +59,10 @@ export interface AccountPreviewProps {
    * @default false
    */
   open?: boolean;
+  /**
+   * To override the default `sx` value on the `Stack` component in the "expanded" variant
+   */
+  sx?: SxProps;
```

Comment:
So it applies to the expanded variant only? Probably ideally it would apply to any variant to be less confusing.
If you're not using it yet maybe we can skip this prop for now?

Reply:
Fair enough, we can leave it out for now, adding it will not be a breaking change later